sda-commons-starter-web/src/main/java/org/sdase/commons/spring/boot/web/client/OidcClientRequestConfiguration.java

```diff
@@ -29,35 +30,44 @@ public OidcClientRequestConfiguration(ApplicationContext applicationContext) {
   @Bean
   @Lazy
   @ConditionalOnProperty(value = "oidc.client.enabled", havingValue = "true")
-  public RequestInterceptor getOidcRequestInterceptor() {
-    return new OidcClientRequestInterceptor(applicationContext);
+  public RequestInterceptor getOidcRequestInterceptor(
+      @Value("${oidc.client.token-pass-through.enabled:true}") boolean authenticationPassthrough) {
```

Comment:
Could you please align the property name? -> `oidc.client.authentication-passthrough.enabled`

Reply:
Sure. Adjusted